ports/dbus-cxx/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO dbus-cxx/dbus-cxx
+    REF "${VERSION}"
+    SHA512 ad6551d03d0c7d499e9f0c6d77584e39d361a1464017be3c40c237d4c43306ad0ffb49b52c06b89cd62ec7346ebcb29f3d166a31b245fd978159e337a08ebafb
+    HEAD_REF master
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        "glib"          ENABLE_GLIB_SUPPORT            # Build dbus-cxx with GLib module
+        "qt5"           ENABLE_QT_SUPPORT              # Build dbus-cxx with QT5 module
+        "libuv"         ENABLE_UV_SUPPORT              # Build dbus-cxx with libuv module

```

These comments are redundant.
```suggestion
        "glib"          ENABLE_GLIB_SUPPORT
        "libuv"         ENABLE_UV_SUPPORT
        "qt5"           ENABLE_QT_SUPPORT
```